spec/system/work_spec.rb

```diff
@@ -68,10 +67,21 @@
       nodeset = doc.xpath("/xmlns:resource")
       expect(nodeset).to be_instance_of(Nokogiri::XML::NodeSet)
     end
+  end
+
+  context "invalid datacite record" do
+    let(:work) { FactoryBot.create :draft_work }
+    let(:invalid_xml) { file_fixture("datacite_basic.xml").read.gsub("<creator", "<invalid") }
+
+    before do
+      stub_s3
+      sign_in user
+      allow_any_instance_of(PDCMetadata::Resource).to receive(:to_xml).and_return(invalid_xml)
+    end
 
     it "Validates the record and prints any errors", js: true do
       visit datacite_validate_work_path(work)
-      expect(page).to have_content "The value has a length of '0'"
+      expect(page).to have_content "This element is not expected"
     end
   end
```

Comment:
Passing an invalid XML to serialize is now more difficult since we always use our internal `@work.resource` object and that object is serialized to XML via the PDCSerialization::Datacite. The new serialization rejects on creation any object that does not meet the required fields and therefore I had to change the test to force it to return an invalid XML so the code in the controller that does the validation is executed and tested.